src/main/scala/uk/gov/hmrc/ct/accounts/formats/CompanyAddressFormatter.scala

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2016 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.ct.accounts.formats
+
+import play.api.libs.json.Json
+import uk.gov.hmrc.ct.accounts.CompanyAddress
+
+object CompanyAddressFormatter {
+  
+  def fromJsonString(json: String): CompanyAddress = Json.fromJson[CompanyAddress](Json.parse(json)).get
```

Comment:
OK. @pncampbell seems to think we don't need them anymore. I'll merge it but it would good to remove them and see if it still works in filing.